Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 ignored = [
-  "github.com/openshift-metalkube/kni-installer/tests*",
-  "github.com/openshift-metalkube/kni-installer/pkg/terraform/exec*"
+  "github.com/openshift-metal3/kni-installer/tests*",
+  "github.com/openshift-metal3/kni-installer/pkg/terraform/exec*"
```

Comment:
Did this help? openshift-metalkube/kni-installer should redirect to the repo, and until now I think everything in here is pointing at the old name

Reply:
No. Now "dep ensure" panics after 30+ minutes. https://gist.github.com/dhellmann/3931a0e1d871ffdbf9df601f1a1fafcf

"dep check" reports a ton of errors, most of them related to parts of the installer importing other parts using the old name: https://gist.github.com/dhellmann/bf663f9a51fc2716d8d649cbb89cbf42

Reply:
Oh, do you have the installer checked out into `$GOPATH/src/openshift-metal3`? It should be checked out to `openshift-metalkube/kni-installer` until the rename goes through, or we settle on reverting all the name changes alltgoether (#66)